src/main/java/com/company/managementservice/model/dto/EmployeeDto.java

```diff
@@ -0,0 +1,56 @@
+package com.company.managementservice.model.dto;
+
+import com.company.managementservice.model.entity.Employee;
+import com.company.managementservice.model.enums.DesignationType;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class EmployeeDto {
+
+    @JsonProperty("firstName")
+    private String firstName;
+
+    @JsonProperty("lastName")
+    private String lastName;
+
+    @NotNull
+    @JsonProperty("emailId")
+    private String emailId;
+
+    @NotNull
+    @JsonProperty("address")
+    private String address;
+
+    //designation ENUMS,  status ENUM*
```

Comment:
Add Enums

Reply:
Designation enum is present, status is is_terminated only